src/platform/middleware.ts

```diff
@@ -73,3 +75,16 @@ export const gracefullyDenyDeletedDevices: RequestHandler = (
 			next();
 		});
 };
+
+export const registerDeviceContact = (path: string) => {
```

Comment:
This would be better as
```suggestion
export const registerDeviceContact = (path: _.PropertyPath) => {
```